includes/compiler.php

```diff
@@ -219,15 +219,15 @@ function checkForDependencies() {
 	// Here, we're going to see if we have the functions that we need. :D
 	$needed = array();
 	// First things first:
-	if(version_compare(PHP_VERSION, '5.3.0', '<')) {
+	if(version_compare(PHP_VERSION, '7.0.0', '<')) {
```

Comment:
This makes >= PHP 7.0 a requirement. Considering what I assume to be the very limited and somewhat legacy user base (for good reason) of this version of THT this change would hurt adoption even further. A requirement of PHP 7 (or 7.1, really) for THT2 makes sense but here I'm not so sure. 5.6 would probably be the best tradeoff.